### PR TITLE
Add Portuguese translation

### PR DESCRIPTION
**Pull request by Vitor238**
Translation for use of the library in Portuguese

### CODE REVIEW

**Comment by joielechong** [comment]
Thank you very much!

**Comment by joielechong** [comment]
Can you check for the conflicted string? I have zero capability on Portuguese :(

File: countrycodepicker/src/main/res/values-pt/strings_country.xml [hunk 1]
```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="country_afghanistan_name">Afeganistão</string>
```
Comment (zeno-mir) [inline countrycodepicker/src/main/res/values-pt/strings_country.xml:3]: I think this line takes care of showing the country names in a given language: https://github.com/joielechong/CountryCodePicker/blob/master/countrycodepicker/src/main/java/com/rilixtech/widget/countrycodepicker/CountryCodeArrayAdapter.java#L78

Also see: https://developer.android.com/reference/java/util/Locale#getDisplayCountry()

So we don't need to add `strings_country.xml` files for any language

File: countrycodepicker/src/main/res/values-pt/strings.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="search_hint">Buscar &#8230;</string>
+  <string name="no_result_found">Nenhum resultado foi encontrado</string>
+  <string name="select_country">Selecionar país</string>
```
Comment (zeno-mir) [inline countrycodepicker/src/main/res/values-pt/strings.xml:5]: I personally think these are better than what we added in https://github.com/joielechong/CountryCodePicker/pull/92/commits/faeacd58100e68df92fb49ee7aa0b4671c2d4482
But then, I don't speak Portuguese, just a tiny bit of Spanish 😉

**Comment by joielechong** [comment]
Thanks again @Vitor238 !

**Pull request merged**